src/mlock.rs

```diff
@@ -2,20 +2,50 @@
 
 #![cfg(feature = "use_os")]
 
+use libc::_SC_PAGESIZE;
+
+/// Retrieve page size of the system
+///
+/// Used for alignment
+unsafe fn page_size() -> usize {
```

Comment:
We can use the page size cache in alloc mod.

Reply:
You're right, shouldn't be doing syscalls everytime we mlock/munlock. I've initialized the `alloc_init()` function and returned the `PAGE_SIZE` from the `alloc` module instead